src/app/pages/mentor-details/mentor-details.page.html

```diff
@@ -1,5 +1,5 @@
 <!-- <app-page-header [config]="headerConfig"></app-page-header> -->
-<app-generic-profile-header [headerData]='detailData.data' [buttonConfig]="buttonConfig" [showRole]="false">
+<app-generic-profile-header [headerData]='detailData.data' [buttonConfig]="buttonConfig" [mentorId]='mentorId' [showRole]="false">
```

Comment:
Rather than creating a new key as mentorId ,can we send it as some buttonConfig? Else we will end up creating lot of inputs.  Check if you can add a meta to the buttonConfig